robot/control/Inc/MicroPackets.hpp

```diff
@@ -110,7 +110,9 @@ struct RadioError {
     bool initialized = false; /**< Stores whether Radio has been initialized */
     uint32_t lastUpdate;      /**< Time at which RadioError was last updated (milliseconds) */
 
-    bool hasError;            /**< Stores whether Radio has an error */
+    bool hasError;                 /**< Stores whether Radio has an error */
```

Comment:
Can you change the comment to specify it's a logical or of the hasConnectionError/hasSoccerConnectionError? Either that or completely remove this since the other errors are more specific

Reply:
`hasError` is just meant to indicate whether the radio has any error at all. It's basically the or of all other radio errors.